src/main/diagnostics/steps/step11.auth.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {session} from 'electron';
+import {ElectronLog} from 'electron-log';
+import {DiagnosticStepResponse} from 'types/diagnostics';
+
+import DiagnosticsStep from '../DiagnosticStep';
+
+const stepName = 'Step-11';
+const stepDescriptiveName = 'AuthSSO';
+
+const run = async (logger: ElectronLog): Promise<DiagnosticStepResponse> => {
+    try {
+        logger.debug(`Diagnostics ${stepName} run`);
+        const cookies = await session.defaultSession.cookies.get({});
```

Comment:
We might want to sanitize some of the cookies in some cases when we're sending stuff around, since they will contain reusable tokens. Is this option available?